src/coreclr/vm/assemblynative.cpp

```diff
@@ -1407,3 +1409,57 @@ void QCALLTYPE AssemblyNative::TraceSatelliteSubdirectoryPathProbed(LPCWSTR file
 
     END_QCALL;
 }
+
+// static
+INT32 QCALLTYPE AssemblyNative::ApplyHotReloadUpdate(
+    QCall::AssemblyHandle assembly,
+    UINT8* metadataDelta,
+    INT32 metadataDeltaLength,
+    UINT8* ilDelta,
+    INT32 ilDeltaLength,
+    UINT8* pdbDelta,
+    INT32 pdbDeltaLength)
+{
+    QCALL_CONTRACT;
+
+    INT32 result = E_NOTIMPL;
+
+    BEGIN_QCALL;
+
+    _ASSERTE(assembly != nullptr);
+    _ASSERTE(metadataDelta != nullptr);
+    _ASSERTE(metadataDeltaLength > 0);
+    _ASSERTE(ilDelta != nullptr);
+    _ASSERTE(ilDeltaLength > 0);
+
+#ifdef EnC_SUPPORTED
+    GCX_COOP();
+    {
+        if (!CORDebuggerAttached())
+        {
+            // Suspend the runtime.
```

Comment:
Why does this need to suspend the runtime? I think it should be possible to apply the delta without runtime suspended.

Reply:
Ok, I'll remove it. I add this in anticipation of the batching/multi-assembly changes discussed in the issue and never removed it.